src/components/Navbar/styles.js

```diff
@@ -0,0 +1,40 @@
+import { makeStyles } from '@material-ui/core'
+import { colors } from '../../styles/colors'
+
+const useStyles=makeStyles(theme=>({
+    header: {
+        backgroundColor:colors.blue,
+        paddingRight: "8rem",
+        paddingLeft: "6rem",
+        "@media (max-width: 900px)": {
+          paddingLeft: 0,
+        },
+      },
+      logo: {
+        width:'10%',
```

Comment:
width is getting too low during different screen size, I guess we can have some `min-width` for that.